tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
+
     /* Basic Options */
-    "target": "ES5",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "target": "ES6",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```

Comment:
minor: comment doesn't match the chosen target option :) 

(do we need these comments at all?)

Reply:
I removed the comments, they're probably outdated due to ts updates anyway.